envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -66,6 +66,7 @@ typealias Client = String
 class IncomingPermissionsProperties {
     var enabled = false
     var clientIdentityHeaders = listOf("x-service-name")
+    val requestIdentificationHeaders = listOf("x-custom-request-id")
```

Comment:
In docs I can see that deafult is `[ x-request-id ]` but here we have `x-custom-request-id`.